e2e/configmap.go

```diff
@@ -116,5 +116,5 @@ func createCustomConfigMap(c kubernetes.Interface, pluginPath string, subvolgrpI
 	cm.Namespace = cephCSINamespace
 	// since a configmap is already created, update the existing configmap
 	_, err = c.CoreV1().ConfigMaps(cephCSINamespace).Update(context.TODO(), &cm, metav1.UpdateOptions{})
-	return err
+	return fmt.Errorf("failed to update configmap: %w", err)
```

Comment:
You will need to change this to

```go
if err != nil {
    return fmt.Errorf("failed to update configmap: %w", err)
}
return nil
```

This is what the e2e jobs cause to fail with:
```
Jun  3 09:32:57.401: failed to create configmap with error failed to update configmap: %!w(<nil>)
```

Reply:
okay got that.

Reply:
@nixpanic when update fails, do we really get any error ?

Reply:
@humblec yes, after updating the code as mentioned above, we will get an error. Earlier even if the error was nil due to fmt.Errorf the create configmap was failing.

Reply:
hmmm.. some object update errors are ignored from api server iic, any way does not hurt us to have the error return here.